src/helpers/transaction.js

```diff
@@ -40,7 +40,12 @@ export default class Transaction {
 			if (SYSTEMS.DATA.ITEM_TRANSFORMER && !remove) {
 				itemData = await SYSTEMS.DATA.ITEM_TRANSFORMER(itemData);
 			}
-			const incomingQuantity = Math.abs(data.quantity ?? Utilities.getItemQuantity(itemData)) * (remove ? -1 : 1);
+			const incomingQuantity = set
+				? Math.abs(data.quantity ?? Utilities.getItemQuantity(itemData))
+				: Math.abs(data.quantity ?? Utilities.getItemQuantity(itemData)) * (remove ? -1 : 1);
+			const incomingCost = set 

```

Comment:
I'm a bit confused with this - wouldn't the cost of the item become negative because it was removed from an actor?

Reply:
The reasoning in my head was "are we sure there is no system that handles negative price values on items in this way?", but looking at it again actually no makes sense... I will remove it.

Reply:
> The reasoning in my head was "are we sure there is no system that handles negative price values on items in this way?", but looking at it again actually no makes sense... I will remove it.

rmeove "remove" for the cost